<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>800</height>
   </rect>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="qt::toolkit::TableView" name="controllerTableView"/>
      <widget class="connectionMatrix::View" name="routingTableView"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="actionExportFullNetworkState"/>
    </widget>
    <addaction name="menuExport"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionChangeLog"/>
    <addaction name="separator"/>
    <addaction name="actionOpenProjectWebPage"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionMediaClockManagement"/>
    <addaction name="actionDeviceFirmwareUpdate"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="loggerDockWidget">
   <property name="windowTitle">
    <string>Logger</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="LoggerView" name="loggerView"/>
  </widget>
  <widget class="QDockWidget" name="entityInspectorDockWidget">
   <property name="windowTitle">
    <string>Entity Model Inspector</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="EntityInspector" name="entityInspector"/>
  </widget>
  <widget class="QToolBar" name="controllerToolBar">
   <property name="windowTitle">
    <string>Controller Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="utilitiesToolBar">
   <property name="windowTitle">
    <string>Utilities Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionChangeLog">
   <property name="text">
    <string>Change Log...</string>
   </property>
  </action>
  <action name="actionMediaClockManagement">
   <property name="text">
    <string>Media Clock Management...</string>
   </property>
  </action>
  <action name="actionOpenProjectWebPage">
   <property name="text">
    <string>Open Project WebPage</string>
   </property>
  </action>
  <action name="actionExportFullNetworkState">
   <property name="text">
    <string>Full Network State...</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
  <action name="actionDevice_Firmware_Update">
   <property name="text">
    <string>Device Firmware Update...</string>
   </property>
  </action>
  <action name="actionDeviceFirmwareUpdate">
   <property name="text">
    <string>Device Firmware Update...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>connectionMatrix::View</class>
   <extends>QTableView</extends>
   <header>connectionMatrix/view.hpp</header>
  </customwidget>
  <customwidget>
   <class>LoggerView</class>
   <extends>QWidget</extends>
   <header>loggerView.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qt::toolkit::TableView</class>
   <extends>QTableView</extends>
   <header>toolkit/tableView.hpp</header>
  </customwidget>
  <customwidget>
   <class>EntityInspector</class>
   <extends>QWidget</extends>
   <header>entityInspector.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openSettings()</slot>
 </slots>
</ui>
